File: Arnav Sharma/Assignment-1/frequency.java
package com.array;

public class frequency {
    public static void main(String[] args) {
        int [] arr = new int [] {1, 2, 8, 3, 2, 2, 2, 5, 1};
        int [] fr = new int [arr.length];
        int visited = -1;
        for(int i = 0; i < arr.length; i++){
            int count = 1;
            for(int j = i+1; j < arr.length; j++){
                if(arr[i] == arr[j]){
                    count++;
                    fr[j] = visited;
                }
            }
            if(fr[i] != visited)
                fr[i] = count;
        }

        System.out.println(" Element | Frequency");
        for(int i = 0; i < fr.length; i++){
            if(fr[i] != visited)
                System.out.println("    " + arr[i] + "    |    " + fr[i]);
        }
    }
}

File: akashmartule/Arrays/frequency.cpp
#include <iostream>
#define MAX_SIZE 100
using namespace std;

int arr[MAX_SIZE];
int i,j,n,element,count;

int main()
{
   

    cout<<"Enter the number of elements:"<<endl;
    cin>>n;

    cout<<"Enter "<< n <<" elements in the array:";
    for(i=0;i<n;i++)     
    {
        cin >> arr[i];
    }  

    cout<<"\n"<<"Elements in the array are:"<<"\t";

    for(i=0;i<n;i++)
    {
        cout<<arr[i]<<"  "<<"\t"<<endl;
    }

    cout<<"Enter the element whose frequency is to be calculated:"<<endl;
    cin>>element;

    count=0;
    for(i=0;i<n;i++)
    {
        element = arr[i];
        count++;
    }

    cout<<"Frequency:"<<endl;
    cout<<count<<endl;

    return 0;
}